modules/generic/src/test/scala/effects.scala

```diff
@@ -76,15 +77,13 @@ final class GenericEffectSpec extends CatsSuite {
 
     val prg: IO[(Json, Int)] =
       for {
-        ref  <- SignallingRef[IO, Int](0)
-        map  =  new GenericEffectMapping(ref)
-        res  <- map.compileAndRun(query)
-        eff  <- ref.get
+        ref <- SignallingRef[IO, Int](0)
```

Comment:
I'd rather not have whitespace only changes.

Reply:
Ah yeah I was wondering whether you have a formatting tool? I applied scalafmt locally since I was too lazy to align things - is there any reason you don't already have it?

Reply:
I'm not a fan of code formatters. Maybe one day.